internify/src/store/actions/studentActions.js

```diff
@@ -1,5 +1,10 @@
-import axios from 'axios';
-import { ADD_STUDENT, REMOVE_STUDENT, UPDATE_STUDENT, GET_STUDENTS} from "./types/studentTypes";
+import axios from "axios";
```

Comment:
can we leave this like it was before, it clogs up the file and makes it too long when we create a new line for each import object @ReneReid  @tommytwm  @aleemer I noticed ppl kept changing my code to use this new format and I don't get why xD

Reply:
i think i know what's happening here - I've been using "Prettify" on all my files almost automatically. I think that was recommended by Tommy - the result is changes in some formatting